spec/lib/pdf_filler/md_el101_pdf_spec.rb

```diff
@@ -6,7 +6,7 @@
   let(:intake) { create(:state_file_md_intake) }
   let(:submission) { create :efile_submission, tax_return: nil, data_source: intake }
   let(:pdf) { described_class.new(submission) }
-  let(:expected_signature_date) { DateTime.now.in_time_zone(StateFile::StateInformationService.timezone('md')).strftime("%F") }
+  let(:expected_signature_date) { DateTime.new(2024, 12, 19, 12).strftime("%F") }
 
```

Comment:
i feel like we still need to use in_time_zone(StateFile::StateInformationService.timezone('md') on the esigned at because its stored in UTC time and then we sign in MD timezone. id40 pdf has a good example i think

Reply:
hmm ok, i can add the timezone. will probably also add to the two line changes below?

Reply:
this approach is just a little different b/c it sets UTC 12/19/2024 12PM as the time, which should translate to "12-19-2024" which ever timezone we're in. I think this approach as-is still makes sense, but happy to chat about it more. timezone stuff is so confusing to me

Reply:
ohhh noon time does make sense, but would it be better to do something like let(:today_noon) { Time.current.noon } instead of hardcoding the date

Reply:
@tahsinaislam just flagging here for you this test failed b/c i had set the signed_at timestamp to 1.hour.ago so there was always a time span where tests would break 😅 anyways see the most recent PR for the way we want to test timestamps going forward!